app/lib/actions/katello/product/create.rb

```diff
@@ -4,6 +4,8 @@ module Product
       class Create < Actions::EntryAction
         def plan(product, organization, subscription_start = nil)
           sequence do
+            ::Katello::Repository.ensure_sync_notification
```

Comment:
In this instance it likely doesn't matter too much because of the nature of the call but i'll move it to run regardless